model/model.go

```diff
@@ -98,14 +99,34 @@ type Container struct {
 	Ports        []ExposedPort `json:"ports" yaml:"ports"`
 	MemoryLimit  string        `json:"memoryLimit,omitempty" yaml:"memoryLimit,omitempty"`
 	MountSources bool          `json:"mountSources" yaml:"mountSources"`
+
+	// Base root command inside container
+	Command []string `json:"command" yaml:"command"`
```

Comment:
What is the goal of this addition and how is it managed by the plugin broker ?

Reply:
Command and args defines root process entrypoint container command. We could declare here command, for example,  to copy remote binary from container to volume. Currently plugin broker doesn't manage it, this logic was implemented in the  che-server...